examples/hpc-enterprise-slurm.yaml

```diff
@@ -142,7 +142,7 @@ deployment_groups:
         family: $(vars.family)
         project: $(vars.project)
       instance_image_custom: true
-      bandwidth_tier: tier_1_enabled
+      bandwidth_tier: 'tier_1_enabled'
```

Comment:
Remove the quotes and just leave it as it was.

Reply:
Done.